From f0f0afd2b0df3574a36e14f2ec71524f4c28eab5 Mon Sep 17 00:00:00 2001
From: Marvin Scholz <epirat07@gmail.com>
Date: Thu, 4 Nov 2021 16:15:53 +0100
Subject: [PATCH] Add EXE_EXT to allow specifying binary extension

This is needed to make the install step aware that the lua/luac
binaries have a .exe suffix for Windows.
---
 Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3e327ee..80cc1d6 100644
--- a/Makefile
+++ b/Makefile
@@ -34,13 +34,16 @@ INSTALL_DATA= $(INSTALL) -m 0644
 MKDIR= mkdir -p
 RM= rm -f
 
+# File extension used for executables
+EXE_EXT=
+
 # == END OF USER SETTINGS -- NO NEED TO CHANGE ANYTHING BELOW THIS LINE =======
 
 # Convenience platforms targets.
 PLATS= guess aix bsd c89 freebsd generic linux linux-readline macosx mingw posix solaris
 
 # What to install.
-TO_BIN= lua luac
+TO_BIN= lua$(EXE_EXT) luac$(EXE_EXT)
 TO_INC= lua.h luaconf.h lualib.h lauxlib.h lua.hpp
 TO_LIB= liblua5.4.a
 TO_MAN= lua.1 luac.1
-- 
2.32.0 (Apple Git-132)

